modules/protect.php

```diff
@@ -538,10 +540,13 @@ function kill_login() {
 	 * Checks if the protect API call has failed, and if so initiates the math captcha fallback.
 	 */
 	public function check_use_math() {
-		$use_math = $this->get_transient( 'brute_use_math' );
-		if ( $use_math ) {
-			include_once dirname( __FILE__ ) . '/protect/math-fallback.php';
-			new Jetpack_Protect_Math_Authenticate;
+		global $pagenow;
+		if ( isset( $pagenow ) && 'wp-login.php' == $pagenow ) {
```

Comment:
@lezama has a fix where this function only gets call on the the login_init action:
#4271

I think that should be good enough. cc @samhotchkiss for a double check.